Documentation/articles/getting_started/4_adding_content.md

```diff
@@ -103,5 +103,3 @@ protected override void Draw(GameTime gameTime)
 Now run the game and you should get the following:
 
 ![Game](~/images/getting_started/3_game.png)
-
-For the next part, look at [Adding Basic Code](4_adding_basic_code.md) page.
```

Comment:
Same as before should allow the user to flow from one getting started section to the next

Reply:
I thought it was redundant to do so as you can just click on the sidebar.

Do our docs really need us to provide Next/Prev on the bottom of each page, if so we shouldn't be adding it by hand.